packages/connect/src/api/getAccountInfo.ts

```diff
@@ -317,18 +311,16 @@ export default class GetAccountInfo extends AbstractMethod<'getAccountInfo', Req
     }
 
     async discover(request: Request) {
-        const { coinInfo, identity, defaultAccountType } = request;
+        const { coinInfo, identity, defaultAccountType, derivationType } = request;
         const blockchain = await initBlockchain(coinInfo, this.postMessage, identity);
         const dfd = this.createUiPromise(UI.RECEIVE_ACCOUNT);
 
         const discovery = new Discovery({
             blockchain,
-            commands: this.device.getCommands(),
-            derivationType:
-                typeof request.derivationType !== 'undefined'
-                    ? request.derivationType
-                    : PROTO.CardanoDerivationType.ICARUS_TREZOR,
```

Comment:
this was totaly useles since `derivationType` default is set from AbstractMethod?

Reply:
No, but every single caller was defaulting to `ICARUS_TREZOR` while the `getAccountDescriptor` command itself throws when received undefined (which never happened). So I set it as a default directly in `getAccountDescriptor` :man_shrugging: